### ISSUE TITLE
Sweeps which were acquired in V-Clamp have some amplifier entries missing

### ISSUE THREAD

**Issue by t-b** [AllenInstitute/MIES#376]
Bug report:

> there's a new NWB file on your ftp server at NWB test data for bugfix/2019_10_02/2019_10_02_163453-compressed.nwb. If you look at sweep 71/AD0 (and a few others) you should find that the recording is in VC but has no VC holding potential in the lab notebook. I think we had encountered this issue long ago and you made a modification to increase logging in the hopes of tracking down the cause?

Looking at the relevant Labnotebook entries reveals the following (sweep 71, HS1):

![grafik](https://user-images.githubusercontent.com/1185677/68979494-04182400-07fe-11ea-9085-c1c830212b65.png)

"Operating Mode" is the clamp mode as read out by `AxonTelegraphGetDataStruct`, "Clamp Mode" is what we think in MIES the clamp mode is. So these two agree and say VC. But we don't have a "V-Clamp Holding Enable" entry. And that means that we took the elseif branch in 

https://github.com/AllenInstitute/MIES/blob/403c8ec284fa87095c27ea212cf91afc33d68e0f/Packages/MIES/MIES_AmplifierInteraction.ipf#L1458-L1490

And that means that the variable `clampMode` was not `V_CLAMP_MODE` (aka zero) but either I_CLAMP_MODE` or `I_EQUAL_ZERO_MODE`.

The labnotebook entry "Clamp Mode" is added to the sweep settings wave in 

https://github.com/AllenInstitute/MIES/blob/403c8ec284fa87095c27ea212cf91afc33d68e0f/Packages/MIES/MIES_DataConfiguratorITC.ipf#L1338

which is *before* the sweep.

TL;DR

The clamp mode changed between the begin and end of the sweep. This could be the case if the user changed the clamp mode in the MCC app instead of in MIES. I've just tried that in the referenced PR and could reproduce the issue in that way.
